crates/ruff_dev/src/generate_cli_help.rs

```diff
@@ -114,12 +114,15 @@ pub(super) fn main(args: &Args) -> Result<()> {
 
 /// Returns the output of `ruff help`.
 fn help_text() -> String {
-    args::Args::command().render_help().to_string()
+    args::Args::command()
+        .term_width(79)
+        .render_help()
+        .to_string()
 }
 
 /// Returns the output of a given subcommand (e.g., `ruff help check`).
 fn subcommand_help_text(subcommand: &str) -> Result<String> {
-    let mut cmd = args::Args::command();
+    let mut cmd = args::Args::command().term_width(79);
```

Comment:
Andrew's abiding by PEP-8, clearly: https://peps.python.org/pep-0008/#maximum-line-length

Reply:
But it's not python ;)

Reply:
oh

Reply:
Anyway, I don't think it matters because markdown is not whitespace sensitive, so I don't think the rendering  changes on the website

Reply:
Yeah Markdown won't care. I've been using 79 columns since forever, so I just wrote that here.

(The actual reason why I use a somewhat short line width more generally is so I can have code windows side-by-side, at a reasonable font size, without auto-line-wrapping kicking in.)